osidb/tests/conftest.py

```diff
@@ -136,3 +136,14 @@ def private_source():
 @pytest.fixture
 def both_source():
     return FlawSource.GENTOO
+
+
+@pytest.fixture(autouse=True)
+def mock_taskman(monkeypatch):
+    def mock_create_or_update_task(*args, **kwargs):
+        pass
+
+    monkeypatch.setattr(
+        "apps.taskman.service.JiraTaskmanQuerier.create_or_update_task",
+        mock_create_or_update_task,
+    )
```

Comment:
This is dangerous and antithetical to testing, if someone makes a change in the osidb module that would somehow break the calls to `create_or_update_task` we might not see it (depends on a lot of factors).